src/styles/card.scss

```
@@ -0,0 +1,99 @@
+$main-color: #060b35;
+
+
+.card {
+  width: 198px;
+  height: 406px;
+  background-color: #fff;
+  border-radius: 5px;
+  box-shadow: 0 4px 10px rgba(0, 0, 0, 0.1);
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    border-radius: 10px;
+    margin: 32px 19px 40px;
+    display: block;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 16px 4px;
+    color: $main-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+    margin: 0 16px 16px;
+  }
+
+  &__rating {
+    display: flex;
+    margin: 0 16px 24px;
+    align-items: center;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $main-color;
+    margin: 0;
+    margin-left: auto;
+  }
+
+  &__price-block {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 16px 16px;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    font-weight: 400;
+    color: #616070;
+    margin: 0;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 800;
+    color: $main-color;
+    margin: 0;
+    margin-left: auto;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    background-color: #00acdc;
+    border-radius: 5px;
+    transition: all 0.3s ease;
+    margin: 0 16px 16px;
+    padding: 0;
+
+    color: #fff;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    text-decoration: none;
+    border: none;
+
+    &:hover {
+      background-color: white;
+      color: #00acdc;
+      border: solid #00acdc;
+    }
+  }
```

You still have the effect of the button becoming larger on hover

you can handle the hover effect like this (leave the border alone :) )

```suggestion
    border: 1px solid 00acdc;

    &:hover {
      background-color: white;
      color: #00acdc;
    }
  }
```
The only thing that remains is making sure the width and height are ok